docs/en/ingest-management/release-notes/release-notes-8.17.asciidoc

```diff
@@ -22,6 +23,39 @@ Also see:
 * {kibana-ref}/release-notes.html[{kib} release notes]
 * {beats-ref}/release-notes.html[{beats} release notes]
 
+// begin 8.17.2 relnotes
+
+[[release-notes-8.17.2]]
+== {fleet} and {agent} 8.17.2
+
+Review important information about the {fleet} and {agent} 8.17.2 release.
+
+[discrete]
+[[security-updates-8.17.2]]
+=== Security updates
+
+{fleet-server}::
+* Upgrade `golang.org/x/net` to v0.34.0 and `golang.org/x/crypto` to v0.32.0. {fleet-server-pull}44405[#4405]
+
+{agent}::
+* Update Go version to 1.22.11. {agent-pull}6686[#6686]
```

Comment:
```suggestion
```

This is an illusion, https://github.com/elastic/elastic-agent/commit/72f4328cd2edbba1d28002dda0676087fd8555c1 is incomplete and doesn't bump the .go-version file which is the only one that matters :)